source/data-api/authenticate.txt

```diff
@@ -0,0 +1,152 @@
+
+.. note::
+   
+   You must :ref:`enable an authentication provider
+   <authentication-providers>` before users can authenticate with
+   it.
+
+This guide shows you how to include authentication information in Data
+API requests.
+
+.. _data-api-credential-headers:
+
+Credential Headers
+------------------
+
+You can authenticate a Data API request by including the user's login
+credentials in the request headers. The exact headers to include depend
+on the authentication provider.
+
+Credential headers are useful for:
+
+- requests sent from a server-side application
+
+- requests sent from a command-line tool
+
+- manual or test requests sent from an HTTPS client like Postman
+
+You cannot use credential headers to authenticate requests sent from a
+web browser due to :wikipedia:`Cross-Origin Resource Sharing
+<Cross-origin_resource_sharing>` restrictions. Instead, use
+:ref:`Bearer Authentication <data-api-bearer-authentication>`.
+
+Email/Password
+~~~~~~~~~~~~~~
+
+To authenticate a Data API request as an :ref:`email/password
+<email-password-authentication>` user, include the user's credentials in
+the request's ``email`` and ``password`` headers.
+
+.. code-block:: shell
+   :emphasize-lines: 2-3
+
+   curl -X POST 'https://data.mongodb-api.com/app/<AppID>/endpoint/data/v1/action/find' \
+      --header 'email: <EmailAddress>' \
+      --header 'password: <Password>' \
+      --header 'Content-Type: application/json' \
+      --data-raw '{
+        "dataSource": "mongodb-atlas",
+        "database": "sample_mflix",
+        "collection": "movies",
+        "filter": {
+          "title": "The Matrix"
+        }
+      }'
+
+API Key
+~~~~~~~
+
+To authenticate a Data API request with an :ref:`API Key
+<api-key-authentication>`, include the API key in the request's
+``apiKey`` header.
+
+.. code-block:: shell
+   :emphasize-lines: 2
+
+   curl -X POST 'https://data.mongodb-api.com/app/<AppID>/endpoint/data/v1/action/find' \
+      --header 'apiKey: <APIKey>' \
+      --header 'Content-Type: application/json' \
+      --data-raw '{
+        "dataSource": "mongodb-atlas",
+        "database": "sample_mflix",
+        "collection": "movies",
+        "filter": {
+          "title": "The Matrix"
+        }
+      }'
+
+Custom JWT
+~~~~~~~~~~
+
+To authenticate a Data API request as a :ref:`Custom JWT
+<custom-jwt-authentication>` user, include the JWT string in the
+request's ``jwtTokenString`` header.
+
+.. code-block:: shell
+   :emphasize-lines: 2
+
+   curl -X POST 'https://data.mongodb-api.com/app/<AppID>/endpoint/data/v1/action/find' \
+      --header 'jwtTokenString: <JWT>' \
+      --header 'Content-Type: application/json' \
+      --data-raw '{
+        "dataSource": "mongodb-atlas",
+        "database": "sample_mflix",
+        "collection": "movies",
+        "filter": {
+          "title": "The Matrix"
+        }
+      }'
+
+.. _data-api-bearer-authentication:
+
+Bearer Authentication
+---------------------
```

Comment:
imo the bearer authentication docs should be above the credential header auth docs. 

speculation, but i imagine this'll be more useful to folks using the Data API from the client side, which is most ppl.